sdks/python/apache_beam/io/mongodbio.py

```diff
@@ -194,18 +225,72 @@ def display_data(self):
     res['mongo_client_spec'] = self.spec
     return res
 
-  def _get_avg_document_size(self):
+  def _get_split_keys(self, desired_chunk_size, start_pos, end_pos):
+    # if desired chunk size smaller than 1mb, use mongodb default split size of
+    # 1mb
+    if desired_chunk_size < 1:
```

Comment:
What are the units of desired_chunk_size? The comment implies mb?

Reply:
yes in Mb. renamed the argument for clarity.